### ISSUE TITLE
Additional language paths into Cromwell service/run modes

### ISSUE THREAD

**Issue by cjllanwarne** [broadinstitute/cromwell#2313]
E.g. come up with a sensible a way to submit WDL draft 2 files, WDL 1.0 files, CWL files, etc.

DoD: 
 - I can run cromwell in server mode and be able to submit both a WDL file and a CWL files. 
 - It's fine for now that the Materialization actor just throws at the first sight of a CWL input.

**Comment by cjllanwarne** [comment]
Please ping @geoffjentry before picking up this ticket, he has some ideas

**Comment by geoffjentry** [comment]
What I was going to suggest was looking at [WES](https://github.com/ga4gh/workflow-execution-schemas). 

- Conceptually it's what we need to do anyways ("Hey, here's a workflow and it's of type X")
- We've signed on to support WES at a future date anyways

It's possible that it doesn't map particularly cleanly and/or doesn't make sense for some other reason but IMO it'd be good to at least poke at going down this path first. There's protobuf & swagger in that repo I linked.

If there are complaints/feedback about the API and not the Cromwell implications of the API, let me know and we can surface them w/ the appropriate folks.
